test/eventing.bash

```diff
@@ -29,70 +29,6 @@ function prepare_knative_eventing_tests {
   logger.info 'Nothing to prepare for Eventing upgrade tests'
 }
 
-function run_eventing_preupgrade_test {
```

Comment:
Why all the changes to the eventing scripts :thinking: ?

Reply:
Martin, you are removing here the eventing upgrade tests, but you didn't add their new `Operation` to new test.

Reply:
/hold

Reply:
Yes. I know. I was only working on Serving upgrade tests and preparing the scripts for the new upgrade framework. The Eventing tests don't work anyway so the Eventing stuff in scripts can't be used. I was hoping we could do this: merge this for Serving, then the master branch would be running only Serving upgrade tests for a short while until I add the Eventing tests which also depend on the Eventing update to 0.20 (in a separate PR from Matthias). The other way would be also rebasing this on Matthias' PR and add Eventing tests in one go but it's too many dependencies in one PR. This would only work if you review and "stamp" the PR for eventing and I'd include the commits in this PR. WDYT?

Reply:
IMO it's okay to ship piecemeal in this case, but regarding our process discussions we really need to avoid such situations in the future.

Reply:
I'm okay with your original plan, but please add a comment to the code, something like:

```
TODO(mgencur): Restore eventing upgrade tests shortly
```

Reply:
Thanks. Pushed the comment.